sdcflows/utils/wrangler.py

```diff
@@ -540,7 +540,7 @@ def find_anatomical_estimators(
     base_entities : :class:`dict`
         Entities to use to query for images. These should include any filters.
     suffixes : :class:`list`
-        EPI suffixes, for example ``["bold", "dwi"]``. Associated ``"sbref"``\s
+        EPI suffixes, for example ``["bold", "dwi", "asl"]``. Associated ``"sbref"``\s
         will be found and used in place of BOLD/diffusion EPIs.
```

Comment:
Would this be true?
```suggestion
        EPI suffixes, for example ``["bold", "dwi", "asl"]``. Associated ``"sbref"``\s
        will be found and used in place of BOLD/diffusion EPIs.
        Similarly, ``"m0scan"``\s associated with ASL runs with the ``IntendedFor`` or
        ``B0FieldIdentifier`` metadata will be used in place of ASL runs.
```

Reply:
Ah, great catch. That's definitely true.